web/libs/datamanager/src/sdk/lsf-sdk.js

```diff
@@ -642,19 +665,22 @@ export class LSFWrapper {
         {
           body: serializedAnnotation,
         },
+        // errors are displayed by "toast" event - we don't want to show blocking modal
+        { errorHandler: () => true }
       );
     });
     const status = result?.$meta?.status;
 
-    if (status === 200 || status === 201)
-      this.datamanager.invoke("toast", { message: "Annotation updated successfully", type: "info" });
-    else if (status !== undefined)
-      this.datamanager.invoke("toast", { message: "There was an error updating your Annotation", type: "error" });
+    this.showOperationToast(status, "Annotation updated successfully", "Annotation is not updated", result);
 
     this.datamanager.invoke("updateAnnotation", ls, annotation, result);
 
     if (exitStream) return this.exitStream();
 
+    if (status >= 400 && status < 600) {
+      return;
+    }
```

Comment:
just the first half of condition is enough for http codes
```suggestion
    if (status >= 400) return;
```